tests/product/base_product_case.py

```diff
@@ -417,7 +424,7 @@ def replace_keywords(self, text, cluster=None, **kwargs):
             cluster = self.cluster
         test_keywords = {
             'rpm': self.presto_rpm_filename,
-            'rpm_basename': self.presto_rpm_filename[:-4],
+            'rpm_basename': self.presto_rpm_filename[:-4] + '.*',
```

Comment:
what's the rpm_filename/rpm_basename supposed to represent?  I'm confused why you would add a .* after you get rid of the .rpm from the rpm_filename.